src/docs/components/tagModal/README.md

```diff
@@ -0,0 +1,49 @@
+# Tag Count
+
+This component is used to show the number of tags on a system
+
+## Usage
+
+Import TagCount from this package.
+
+```JSX
+import React from 'react';
+import { TagCount } from '@redhat-cloud-services/frontend-components';
```

Comment:
You also need to show import of the `TagModal` since you're using it below in the sample component